charts/shoot-core/components/charts/metrics-server/templates/deployment.yaml

```diff
@@ -59,6 +59,18 @@ spec:
         - --tls-cert-file=/srv/metrics-server/tls/tls.crt
         - --tls-private-key-file=/srv/metrics-server/tls/tls.key
         - --v=2
+        readinessProbe:
+          tcpSocket:
+            port: 8443
+          initialDelaySeconds: 0
```

Comment:
Shouldn't we set `initialDelaySeconds` to 5 or 10 to allow metrics-server to properly start serving? Isn't `initialDelaySeconds=0` too optimistic?

Reply:
It should not be an issue because it will be retried after 10 seconds and simply will not serve traffic till then. But I will add some initial delay to give it a few seconds to start-up.